Samples/Notifications/Push/README.md

```diff
@@ -0,0 +1,43 @@
+---
+name: "Push Notifications Sample" 
+page_type: sample
+languages:
+- cpp
+products: 
+- windows-app-sdk
+description: "Demonstrates how to use the windows app SDK Push Notifications APIs from an unpackaged app"
+urlFragment: PushNotifications
+---
+# Push Notifications Sample
+This sample demonstrates:
+1. How an app can use the Windows App SDK to request a WNS Channel URI
+    * The WNS Channel URI uniquely identifies the app for a current user on a specific device, allowing other apps and servers to send notifications to this app.
+2.	How to use the Windows App SDK to register an app for foreground and background activation and handle the activations.
+## Requirements
+* A web API tool (like Postman or fiddler) to send notifications.
+   * The Windows App SDK push notifications APIs provides everything an app needs register for and receive notifications but it's expected that an external server app will be sending the notifications. The web API tool is used to emulate that external server app to send notifications.
```

Comment:
Reword to be more brief and less wordy: The Windows App SDK push notifications APIs provides an API to register for and receive notifications from an external app service. The web API tool is used to emulate the external app service.

Reply:
done